testdata/run_alt_goos_goarch.txt

```diff
@@ -1,7 +1,11 @@
-env GOOS=rubbish
-env GOARCH=myarch
+# Test that the value of GOOS and GOARCH does not affect -run
+
+# Set values that we will never test with
+env GOOS=solaris
```

Comment:
Because we now call `go env` as part of `gobin`, this test needs to be against a valid pair of GOOS/GOARCH